scripts/deploy.js

```diff
@@ -44,11 +44,10 @@ async function main() {
     "https://gateway.pinata.cloud/ipfs/QmZTANcaGBNDkxjgrwYibeSzGwa4nCKKAvCRHf45tK87PV/horse3-metadata.json"
   );
 
-  const winner = await myNFT.doTheRace();
-
-  console.log("The winner is:", Number(winner));
-  const get = await myNFT.getAllNFTs();
-  console.log("GetAll", get);
+  // const winner = await myNFT.doTheRace();
```

Comment:
does not work anymore, since one has to pay.

Reply:
well it should, minting nfts also costs gas but it works here